package.json

```diff
@@ -49,6 +49,9 @@
   "devDependencies": {
     "@commitlint/cli": "8.3.5",
     "@commitlint/config-conventional": "8.3.4",
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
```

Comment:
For now, I added these as a `"devDependency"` to use them on `<Button />` examples.

The work to move it to `dependencies` or `peerDependencies` etc... will come in a [separate PR](https://jira.zopa/browse/ZDS-11).